tests/e2e_logging.py

```diff
@@ -793,6 +799,53 @@ def test_forwarding_frontends(network, args):
     return network
 
 
+@reqs.description("Testing signed queries with escaped queries")
+@reqs.at_least_n_nodes(2)
+def test_signed_escapes(network, args):
```

Comment:
This duplicates a lot of `test_forwarding_frontends`, which could be factored out into something like `def escaped_query_tests(client, url_base)`.

Reply:
Yes, good point.